index.html

```diff
@@ -71,1285 +81,1323 @@
+          "name": "MiniApp Demo",
+          "short_name": "MiniApp",
+          "version_name": "1.0.1",
+          "version_code": 11,
+          "description": "A Simple MiniApp Demo",
+          "icons": [
+            {
+              "src": "common/icons/icon.png",
+              "sizes": "48x48"
+            }
+          ],
+          "min_platform_version": "1.0.0",
+          "pages": [
+            "pages/index/index",
+            "pages/detail/detail"
+          ],
+          "window": {
+            "navigation_bar_text_style": "black",
+            "navigation_bar_title_text": "My MiniApp",
+            "navigation_bar_background_color": "#f8f8f8",
+            "background_color": "#ffffff",
+            "fullscreen": false
+          },
+          "widgets": [
+            {
+              "name": "widget",
+              "path": "widgets/index/index",
+              "min_platform_version": "2.0.0"
+            }
+          ],
+          "req_permissions": [
+            {
+              "name": "system.permission.LOCATION",
+              "reason": "To show user's position on the map"
+            },
+            {
+              "name": "system.permission.CAMERA",
+              "reason": "To scan a QR code"
+            }
+          ]
+        }
+      </pre>
+    </section>
+
     <section id='webappmanifest-members'>
-      <h2>Members inherited from the Web Application Manifest</h2>          
+      <h2>Web Application Manifest members</h2>
+      <p>The [=MiniApp manifest=] uses the following essential [=application manifest=]'s members.</p>      
       <section>
-        <h3>
-          <span its-locale-filter-list="en" lang="en"><code>dir</code> member</span>
-          <span its-locale-filter-list="zh-hans" lang="zh-hans">方向</span>
-        </h3>
-        <p its-locale-filter-list="en" lang="en">
-          <dfn>dir</dfn> member, while
-          specifying the base direction of the <a>localizable members</a> of the manifest, also specifies
-          the default base text direction of the whole MiniApp. The <a>dir</a>
-          member's value can be set to one of the <a href="https://www.w3.org/TR/appmanifest/#dfn-text-direction-values">text-direction values</a> as specified in [[appManifest]].
+        <h3><code>dir</code> member</h3>
+        <p>
+          The [=MiniApp manifest's=] <dfn><code>dir</code></dfn> member, while specifying the base direction of the <a>localizable members</a> of the [=manifest=], also specifies
+          the default base text direction of the whole MiniApp.</p> 
+        <p>The <a><code>dir</code></a> member's value can be set to one of the <a href="https://www.w3.org/TR/appmanifest/#dfn-text-direction-value">text-direction values</a> as specified in [[APPMANIFEST]]. The text-direction values are the following:</p>
+        <dl>
+          <dt><a href="https://w3c.github.io/miniapp-packaging/#dfn-ltr" data-type="dfn" data-link-type="dfn"><code>"ltr"</code></a></dt>
```

Comment:
This link is 404. Same for other values of `dir`.

Reply:
Thanks @xfq , solved at d2c5e02